src/main/antlr/org/hibernate/sqm/parser/hql/internal/antlr/HqlParser.g4

```diff
@@ -66,6 +66,7 @@ targetFieldsSpec
 // ORDER BY clause
 
 orderByClause
+// todo : null precedence
```

Comment:
null-precedence really belongs on `sortSpecification` rule, not `orderByClause`.